kernel/core_hook.c

```diff
@@ -513,13 +513,15 @@ static bool should_umount(struct path *path)
 
 static void ksu_umount_mnt(struct path *path, int flags)
 {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0)
```

Comment:
#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0) || if defined(XXX)

This way it will never break gki.